hedera-node/hedera-app/src/main/java/com/hedera/node/app/grpc/impl/MethodBase.java

```diff
@@ -185,7 +185,7 @@ public void invoke(
             @NonNull final Metrics metrics,
             @NonNull final String nameTemplate,
             @NonNull final String descriptionTemplate) {
-        final var baseName = serviceName + "/" + methodName;
+        final var baseName = serviceName + ":" + methodName;
```

Comment:
Why do we need this ? Because only "/" is not an acceptable character ?

Reply:
Yes. Prometheus is very strict and allows only alphanumeric characters plus ':' and '_'.